web/views.py

```
@@ -6955,3 +6955,51 @@ def delete_post(request, post_id):
     if request.method == "POST":
         post.delete()
     return redirect("social_media_dashboard")
+
+
+def users_list(request):
```

Ruff (0.8.2)

Missing return type annotation for public function `users_list`

(ANN201)